Mathlib/Order/Interval/Set/Basic.lean

```diff
@@ -441,6 +441,9 @@ the equivalence in linear orders, use `Ioi_subset_Ioi_iff`. -/
 @[gcongr]
 theorem Ioi_subset_Ioi (h : a ≤ b) : Ioi b ⊆ Ioi a := fun _ hx => h.trans_lt hx
 
+theorem Ioi_ssubset_Ioi (h : a < b) : Ioi b ⊂ Ioi a :=
+  (ssubset_iff_of_subset (Ioi_subset_Ioi h.le)).mpr ⟨b, h, lt_irrefl b⟩
```

Comment:
Can you add the `Iff` version too, perhaps nearby [`Set.Iio_subset_Iic_iff`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Interval/Set/LinearOrder.html#Set.Iio_subset_Iic_iff)?